plugins/LilyPad/DirectInput.cpp

```diff
@@ -220,7 +222,7 @@ class DirectInputDevice : public Device
             format.dwFlags = 0;
             format.dwNumObjs = numPhysicalControls;
             format.rgodf = formats;
-            int res = did->SetDataFormat(&format);
+            // int res = did->SetDataFormat(&format); // is not used
```

Comment:
Are you sure? `res` doesn't look like it's used anywhere, but `did->SetDataFormat()` seems important (I don't know DirectInput stuff). If this line is commented out, then setting the members of `format` then becomes useless.